### PR TITLE
Add example helm chart, replace example dashboard, add Makefile featu…

### PR DESCRIPTION
**Pull request by jeremyje**
* Fixed Service Account role binding bug in helm chart.
* Added example helm chart.
* Added new Makefile targets for example images.
* Added :dev tag for all images to make config easier for development.

### CODE REVIEW

File: install/helm/open-match/dashboards/grpc.json [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+        "2d",
+        "7d",
+        "30d"
+      ]
+    },
+    "timezone": "",
+    "title": "gRPC",
+    "uid": "nlrmG_Cmk",
+    "version": 4
+  }
```
Comment (sawagh) [inline install/helm/open-match/dashboards/grpc.json]: add a newline at EOF?

File: install/helm/README.md [hunk 2]
```diff
@@ -16,8 +16,9 @@ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admi
...
 # Deploy Open Match
-helm upgrade --install --wait --debug open-match install/helm/open-match \
-	  --namespace=open-match \
-	  --set openmatch.image.registry=gcr.io/open-match-public-images \
-	  --set openmatch.image.tag=0.4.0
+helm upgrade --install --wait --debug open-match \
+    install/helm/open-match \
+    --namespace=open-match \
+    --set openmatch.image.registry=gcr.io/open-match-public-images \
+    --set openmatch.image.tag=0.4.0
```
Comment (sawagh) [inline install/helm/README.md]: --set openmatch.image.registry=$(REGISTRY) \
    --set openmatch.image.tag=$(TAG)

File: Makefile [hunk 3]
```diff
@@ -11,53 +11,55 @@
...
 TOOLCHAIN_DIR = build/toolchain
 TOOLCHAIN_BIN = $(TOOLCHAIN_DIR)/bin
-TOOLCHAIN_GOLANG_DIR = $(TOOLCHAIN_DIR)/golang
 PROTOC := $(TOOLCHAIN_BIN)/protoc
 PROTOC_INCLUDES := $(TOOLCHAIN_DIR)/include/
 GCP_PROJECT_ID = "Set $GCP_PROJECT_ID in your bashrc."
 GCP_PROJECT_FLAG = --project=$(GCP_PROJECT_ID)
 REGISTRY := gcr.io/$(GCP_PROJECT_ID)
 TAG := $(VERSION)
+ALTERNATE_TAG := dev
```
Comment (sawagh) [inline Makefile]: Whats the motivation to have both tags (release and dev)?

Comment (jeremyje) [reply Makefile]: Convenience. Refer to the :dev tag when you want to update new images. the :short_sha images are meant to be immutable.

**Pull request merged**